lib/handlers/user/index.js

```diff
@@ -65,6 +66,19 @@ handlers[EMsg.ClientUpdateMachineAuth] = function(data, callback) {
     }, new schema.CMsgClientUpdateMachineAuthResponse(response).toBuffer());
   });
 };
+handlers[EMsg.ClientNewLoginKey] = function(data) {
+  if(!this._logOnDetails.should_remember_password){
+    return;
+  }
+  var newLoginKey = schema.CMsgClientNewLoginKey.decode(data);
+
+  this._client.send({
```

Comment:
Currently handlers don't send messages on their own, and I would prefer to keep it consistent to avoid surprises. Let the user decide whether to accept the login key.